src/listmode_buildblock/CListModeDataROOT.cxx

```diff
@@ -156,7 +156,7 @@ open_lm_file()
         t.read(mem,4);
         ss << mem;
 
-        if ( !sig.compare(ss.str()) )
+        if (sig.compare(ss.str()) )
```

Comment:
actually, I think we need
```C++
sig.compare(ss.str())!=0
```
Presumably the previous code didn't work due to strange behaviour of ! on the return value of `std::string::compare`

Reply:
This is a very strange error. I never had a problem.